### PR TITLE
Kubernetes image no embedded

### PR DESCRIPTION
**Pull request by potiuk**
The EMBEDDED dags were only really useful for testing
but it required to customise built production image
(run with extra --build-arg flag). This is not needed
as it is better to extend the image instead with FROM
and add dags afterwards. This way you do not have
to rebuild the image while iterating on it.

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: scripts/ci/libraries/_initialization.sh [hunk 1]
```diff
@@ -240,9 +240,9 @@ function get_environment_for_builds_on_ci() {
     export CI_BUILD_ID="0"
     export CI_JOB_ID="default-job-id"
     if [[ ${CI:=} != "true" ]]; then
-        echo
-        echo "This is not a CI environment!. Staying with the defaults"
-        echo
+        print_info
+        print_info "This is not a CI environment!. Staying with the defaults"
```
Comment (michalslowikowski00) [inline scripts/ci/libraries/_initialization.sh]:
```suggestion
        print_info "This is not a CI environment! Staying with the defaults"
```

Comment (michalslowikowski00) [reply scripts/ci/libraries/_initialization.sh]: Should we stick to the interpunction in messages? Sometimes at the end is full stop, sometimes is not.

Comment (michalslowikowski00) [inline scripts/ci/libraries/_initialization.sh]:
```suggestion
        print_info "This is not a CI environment!. Staying with the defaults."
```

**Comment by michalslowikowski00** [comment]
I do not understand 99% of this but this is impressive. :D

**Review by turbaszek — APPROVED** [review]

**Comment by potiuk** [comment]
OK. Merging this one. It has just a transient error on sqlite.

**Pull request merged**